pkg/codegen/nodejs/gen_program_expressions.go

```diff
@@ -411,6 +411,8 @@ func (g *generator) GenLiteralValueExpression(w io.Writer, expr *model.LiteralVa
 	switch expr.Type() {
 	case model.BoolType:
 		g.Fgenf(w, "%v", expr.Value.True())
+	case model.NoneType:
+		g.Fgen(w, "undefined")
```

Comment:
Can we add an issue to track investigating how `None`-typed `LiteralValueExpression`s arise?